tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -155,17 +163,28 @@ private static void handleDescribeReplication(Admin admin) throws ExecutionExcep
         );
     }
 
-    private static List<List<String>> quorumInfoToRows(QuorumInfo.ReplicaState leader, Stream<QuorumInfo.ReplicaState> infos, String status) {
-        return infos.map(info ->
-            Stream.of(
+    private static List<List<String>> quorumInfoToRows(QuorumInfo.ReplicaState leader,
+                                                       Stream<QuorumInfo.ReplicaState> infos,
+                                                       String status,
+                                                       boolean readable) {
+        return infos.map(info -> {
+            final double nowMs = System.currentTimeMillis();
```

Comment:
`System.currentTimeMillis()` returns a long. https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#currentTimeMillis

Using long return type here will makes subtraction easier later on in the code.